docs/guides/volumesnapshot/pvc/index.md

```diff
@@ -135,263 +140,406 @@ spec:
+```bash
+$ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/volumesnapshot/pvc/examples/backupconfiguration.yaml
+backupconfiguration.core.kubestash.com/pvc-volume-snapshot created
+```
 
-- `spec.target.ref`  refers to the backup target. `apiVersion`, `kind` and `name` refers to the `apiVersion`, `kind` and `name` of the targeted workload respectively. Stash will use this information to create a Volume Snapshotter Job for creating VolumeSnapshot.
+**Verify Backup Setup Successful**
 
-- `spec.target.snapshotClassName` indicates the [VolumeSnapshotClass](https://kubernetes.io/docs/concepts/storage/volume-snapshot-classes/) to be used for volume snapshotting.
+If everything goes well, the phase of the `BackupConfiguration` should be in `Ready` state. The `Ready` phase indicates that the backup setup is successful.
 
-Let's create the `BackupConfiguration` crd we have shown above.
+Let's check the `Phase` of the BackupConfiguration,
 
 ```bash
-$ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/volumesnapshot/pvc/examples/backupconfiguration.yaml
-backupconfiguration.stash.appscode.com/pvc-volume-snapshot created
+$ kubectl get backupconfiguration -n demo
+NAME                  PHASE   PAUSED   AGE
+pvc-volume-snapshot   Ready            17s
 ```
 
-**Verify CronJob :**
+**Verify CronJob:**
 
-If everything goes well, Stash will create a `CronJob` to take periodic snapshot of the PVC with the schedule specified in `spec.schedule` field of `BackupConfiguration` crd.
+Verify that KubeStash has created a `CronJob` with the schedule specified in `spec.sessions[*].scheduler.schedule` field of `BackupConfiguration` object.
 
- Check that the `CronJob` has been created using the following command,
+```bash
+$ kubectl get cronjob -n demo
+NAME                                          SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+trigger-pvc-volume-snapshot-frequent-backup   */5 * * * *   False     0        <none>          50s
+```
+
+**Wait for BackupSession:**
+
+Now, wait for the next backup schedule. You can watch for `BackupSession` CR using the following command,
+
+```bash
+$ watch -n 1 kubectl get backupsession -n demo -l=kubestash.com/invoker-name=pvc-volume-snapshot
+
+Every 1.0s: kubectl get backupsession -n demo -l=kubestash.com/invoker-name=pvc-volume-snapshot                                                         anisur: Tue Jan 16 17:01:06 2024
 
- ```bash
- $ kubectl get cronjob -n demo 
-NAME                          SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
-pvc-volume-snapshot           */1 * * * *   False     0        39s             2m41s
- ```
+NAME                                             INVOKER-TYPE          INVOKER-NAME          PHASE       DURATION   AGE
+pvc-volume-snapshot-frequent-backup-1705402801   BackupConfiguration   pvc-volume-snapshot   Succeeded              66s
+```
+
```

Comment:
Here, the phase `Succeeded` means that the backup process has been completed successfully.

Reply:
> applies to all the relevant places